.gitlab/functional_test/regression_detector.yml

```diff
@@ -136,3 +133,43 @@ single-machine-performance-regression_detector:
     - RUST_LOG="${RUST_LOG}" ./smp --team-id ${SMP_AGENT_TEAM_ID} --api-base ${SMP_API} --aws-named-profile ${AWS_NAMED_PROFILE}
       job result
       --submission-metadata submission_metadata
+
+# Shamelessly adapted from golang_deps_commenter job config in
+# golang_deps_diff.yml at commit 01da274032e510d617161cf4e264a53292f44e55.
+single-machine-performance-regression_detector-commenter:
+  stage: functional_test
+  rules:
+    - !reference [.except_main_or_release_branch]
+  image:
+    name: "486234852809.dkr.ecr.us-east-1.amazonaws.com/pr-commenter:3"
+    entrypoint: [""]  # disable entrypoint script for the pr-commenter image
+  tags: ["runner:docker"]
+  needs:
+    - job: single-machine-performance-regression_detector
+      artifacts: true  # overrides default behavior that does not download artifacts
```

Comment:
According to the [documentation](https://docs.gitlab.com/ee/ci/yaml/index.html#needsartifacts) this is the default behaviour, so it shouldn't be necessary.

Reply:
Removed in https://github.com/DataDog/datadog-agent/pull/29785/commits/49a0419691ff29dd4efe3244397f4bde20ada908, in #29785.